src/main.cpp

```diff
@@ -20,6 +20,18 @@ int main()
         sprite.setPosition((float)window.getSize().x / 2, (float)window.getSize().y / 2);
     }
 
+    sf::Font font;
+    if (!font.loadFromFile("arial.ttf"))
+    {
+        return EXIT_FAILURE;
+    }
```

Comment:
I think it would be better to just inform the user that font loading has failed and then let the application run normally without displaying the text.